ui/src/Url.elm

```diff
@@ -36,11 +36,11 @@ routeFromUrl model location =
                     | route = FilteredView product channel platform version locale buildId page
                     , filters =
                         { buildId = buildId
-                        , product = product
-                        , channel = channel
-                        , platform = platform
-                        , version = version
-                        , locale = locale
+                        , product = product |> String.split "|" |> List.filter (\v -> v /= "all")
+                        , channel = channel |> String.split "|" |> List.filter (\v -> v /= "all")
+                        , platform = platform |> String.split "|" |> List.filter (\v -> v /= "all")
+                        , version = version |> String.split "|" |> List.filter (\v -> v /= "all")
+                        , locale = locale |> String.split "|" |> List.filter (\v -> v /= "all")
```

Comment:
I don't agree with this. I think it should be something more like `let parseFilter textFilter = if textFilter == "all" then [] else String.split "|" textFilter`. Right now a URL containing `55.0b1|56.0a1|all` is parsed but I think it shouldn't be.

This might be a nit, because I guess either way, we can't have a filter value called `all`...